tools/windows-node-installer/pkg/openshift/util.go

```diff
@@ -0,0 +1,43 @@
+package openshift
+
+import (
+	"fmt"
+
+	v1 "github.com/openshift/api/config/v1"
+	clientset "github.com/openshift/client-go/config/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Client is an OpenShift clientset struct which will be used for all OpenShift related functions to interact with
+// the existing OpenShift Cluster.
+type Client struct {
+	clientset *clientset.Clientset
+}
+
+// GetClusterID returns the unique identification of the cluster from the given OpenShift Client or an error.
+func (c *Client) GetClusterID() (string, error) {
```

Comment:
Where are we using this function?

Reply:
Then please introduce it when it is actually going to be used.

Reply:
got it,